docs/developers/torch_add_a_new_operator.md

```diff
@@ -0,0 +1,121 @@
+# How To Add a New Torch Operator
+
+![Torch-MLIR Architecture](./pics/Torch-MLIR-MHLO.png)
+
+TorchBlade converts PyTorch workloads to MHLO based on [Torch-MLIR](https://github.com/llvm/torch-mlir/).
+Then compile the MHLO modules via BladeDISC compiler.
+
+The BladeDISC Dev Team is cooperating with the community to add Torch-To-Mhlo
+conversion to Torch-MLIR, especially fully dynamic shape features.
+See RFC: https://github.com/llvm/torch-mlir/issues/999.
+We appeal to the community developers interested in joining.
+
+## To the Developers of Torch-MLIR
+
+TorchBlade Torch operator conversion will comply with Torch-MLIR generally.
+Developers of this part are recommended to have a look at the following:
+
+1. [Torch-MLIR Coding Style](https://github.com/llvm/torch-mlir/wiki/Coding-Style)
+2. [Torch-MLIR Torch Ops E2E Implementation](https://github.com/llvm/torch-mlir/wiki/Torch-ops-E2E-implementation)
+3. [Torch-MLIR Architecture](https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md)
+
+## To the Developers of BladeDISC
+
+### Step 1: Add a New Torch Operator to Supported White List
+
+TorchBlade will cluster supported operation regions into subgraphs and then convert subgraphs from TorchScript
```

Comment:
better to use the present simple.

TorchBlade constructs one or more subgraphs from a TorchScript into MHLO modules via Torch-MLIR, and then BladeDISC compiles these subgraphs into an executable program.